src/redux/books/booksSlice.js

```diff
@@ -1,42 +1,105 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable camelcase */
```

Comment:
- [ ] [OPTIONAL]
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.